h/static/scripts/directive/about-this-version-dialog.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+/**
+ * @ngdoc directive
+ * @name aboutThisVersionDialog
+ * @description Displays product version and environment info
+ */
+// @ngInject
+module.exports = function () {
+  return {
+    bindToController: true,
+    controllerAs: 'vm',
+    // @ngInject
+    controller: function ($scope, $window, settings, crossframe) {
+      this.userAgent = $window.navigator.userAgent;
+      this.version = settings.release;
+      this.dateTime = new Date();
```

Comment:
I don't feel that strongly about it but I didn't see the date mentioned on the card and I'm not sure it will be that useful given that we have the version number.